build.gradle

```diff
@@ -26,15 +26,37 @@ plugins {
 }
 
 project.ext.inspectorImageFamily = "blackduck-imageinspector"
-project.ext.inspectorImageVersion = "5.1.0"
+project.ext.inspectorImageVersion = "5.1.2"
 
-version = '10.1.1-SNAPSHOT'
+version = '10.1.2-SNAPSHOT'
 
 apply plugin: 'org.springframework.boot'
 apply plugin: 'io.spring.dependency-management'
 apply plugin: 'com.bmuschko.docker-remote-api'
 apply plugin: 'com.synopsys.integration.solution'
 
+final def externalRepoHost = "https://sig-repo.synopsys.com"
+final def internalRepoHost = System.getenv('SNPS_INTERNAL_ARTIFACTORY')
+
+repositories {
+    println "Checking if environment property SNPS_INTERNAL_ARTIFACTORY is configured: ${internalRepoHost}"
```

Comment:
Similar question as in the other review: is anything really being "checked" here?